manifests/init.pp

```diff
@@ -30,66 +30,63 @@
 #   means. e.g. environment variable ChocolateyBinRoot.  Defaults to
 #   'C:\tools\chocolatey.server'
 class chocolatey_server (
-  $port = $::chocolatey_server::params::service_port,
-  $server_package_source = $::chocolatey_server::params::server_package_source,
-  $server_install_location = $::chocolatey_server::params::server_install_location,
-) inherits ::chocolatey_server::params {
+  $service_port          = '80',
+  $server_package_source = 'https://chocolatey.org/api/v2/',
+  $server_install_location = 'C:\tools\chocolatey.server',
+  $chocolatey_server_app_pool_name = 'chocolatey.server',
+  $chocolatey_server_app_port      = $port,
+) {
   require chocolatey
 
-  $_chocolatey_server_location      = $server_install_location
-  $_chocolatey_server_app_pool_name = 'chocolatey.server'
-  $_chocolatey_server_app_port      = $port
-  $_server_package_url              = $server_package_source
-
   # package install
   package {'chocolatey.server':
-    ensure    => installed,
-    provider  => chocolatey,
-    source    => $_server_package_url,
+    ensure   => installed,
+    provider => chocolatey,
+    source   => $server_package_source,
   } ->
 
   # add windows features
   windowsfeature { 'Web-WebServer':
   installmanagementtools => true,
   } ->
-  windowsfeature { 'Web-Asp-Net45':
+    windowsfeature { 'Web-Asp-Net45':
   } ->
 
   # remove default web site
-  iis::manage_site {'Default Web Site':
-    ensure        => absent,
-    site_path     => 'any',
-    app_pool      => 'DefaultAppPool'
+  iis_site {'Default Web Site':
+    ensure   => stopped,
+    app_pool => 'DefaultAppPool'
```

Comment:
I am simply because there is a bug which I meant to include. When no sites exist, this creates an issue. https://github.com/voxpupuli/puppet-iis/issues/123